CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 <!-- next version -->
 
+## 0.102.0
+
+### 💡 Enhancements 💡
+
+- `collector`: Add usage metrics for the collector (#2829)
+  This change will add metrics to the OpenTelemetry operator about how the collector is used in the cluster,
+  it will add the following metrics to the opentelemetry-operator metrics endpoint
+  ```
+  opentelemetry_collector_receivers{collector_name="collector_name", namespace="ns", type="otlp"} 1
+  opentelemetry_collector_exporters{collector_name="collector_name", namespace="ns", type="otlp"} 1
+  opentelemetry_collector_processors{collector_name="collector_name", namespace="ns", type="otlp"} 1
+  opentelemetry_collector_connectors{collector_name="collector_name", namespace="ns", type="myconnector"} 0
+  opentelemetry_collector_info{collector_name="simplest",namespace="default", type="deployment"} 1
+  ```
+  
+
+### 🧰 Bug fixes 🧰
+
+- `collector`: Fixes a bug that was preventing regexes from being loaded correctly. Now the filter provide is exactly what's used. (#3007)
+  This is technically a breaking change if a user relied on the previously broken regex functionality.
+  This change will actually fix their regex to work where it didn't before. I expect that users would rather their
+  regexes work than break silently.
+  
+
+### Components
+
+* [OpenTelemetry Collector - v0.102.1](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.102.1)
```

Comment:
It is probably worth calling out that this version fixes the CVE and linking to it.